charms/trusty/clearwater-aio-proxy/lib/reconfigure-aio

```diff
@@ -15,19 +15,27 @@ fi
 old_home_domain=$(. /etc/clearwater/config ; echo $home_domain)
 echo "DELETE FROM ellis.numbers WHERE number LIKE '%@$old_home_domain' AND owner_id IS NULL ;" | mysql
```

Comment:
No.  You can change the number range without changing the home domain.  I did look at working out whether the number range has changed, but that's really difficult, and it didn't seem necessary.  (Destroying/recreating the number pool is pretty cheap/non-invasive).